OWML.Common/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="OW.Unity.Dlls" version="1.0.2" targetFramework="net35" />
+  <package id="Json.Net.Unity3D" version="9.0.1" targetFramework="net35" />
```

Comment:
I wonder why this is needed now and not before. Just commenting this so I can check it again later ;)

Reply:
I think this reference can be removed from most projects